src/components/ui/WorldMap.jsx

```
@@ -24,151 +133,73 @@ export function WorldMap({
-          <img
-            src={`data:image/svg+xml;utf8,${encodeURIComponent(svgMap)}`}
-            className="h-full w-full [mask-image:linear-gradient(to_bottom,transparent,white_10%,white_90%,transparent)] pointer-events-none select-none"
-            alt="world map"
-            height="495"
-            width="1056"
-            draggable={false}
-          />
+          {mapError ? (
+            <div
+              className="h-full w-full flex items-center justify-center"
+              dangerouslySetInnerHTML={{ __html: FALLBACK_SVG }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)